package/kedro_viz/api/graphql.py

```diff
@@ -118,7 +119,7 @@ def format_run_tracking_data(tracking_data: Dict) -> JSONObject:
         >>>         'classWeight":21
         >>>     },
         >>> }
-        >>> format_run_tracking_data(tracking_data)
+        >>> format_run_tracking_data(tracking_data,false)
```

Comment:
```suggestion
        >>> format_run_tracking_data(tracking_data, False)
```
Note `true/false` doesn't work in Python - it needs to be `True/False`. If we give this a default value of `False` then this argument wouldn't be needed here at all though.